Readme.markdown

```diff
@@ -7,17 +7,38 @@ Simple static table views for iOS in Swift. Static's goal is to separate model d
 
 ## Building
 
-Static is written in Swift 2 so Xcode 7b6 is required. There aren't any dependencies besides system frameworks.
+Xcode 7.0 beta 6 is required since Static is written in Swift 2. There aren't any dependencies besides system frameworks.
 
 
 ## Installation
 
+### Carthage
+
 [Carthage](https://github.com/carthage/carthage) is the recommended way to install Static. Add the following to your Cartfile:
 
 ``` ruby
 github "venmo/Static"
 ```
 
+
+### CocoaPods
+
+[CocoaPods](http://cocoapods.org) is a dependency manager for Cocoa projects. To install Static with CocoaPods:
+
+Make sure CocoaPods is installed (Static requires version 0.37 or greater).
+
+Update your Podfile to include the following:
+
+```ruby
+use_frameworks!
+pod 'Static', git: 'https://github.com/venmo/Static'
```

Comment:
I don't think we should be subverting the linter when pushing to trunk. I'm worried that people will pull the pod without knowing about the Xcode 7/Swift 2 requirement and it will cause their `pod install` to fail with vague errors and they'll assume its our fault. Since cocoapods has no way to specify building requirements in the podspec, I think we should keep this in a branch until Xcode 7 is final and cocoapods is updated.